### ISSUE TITLE
root -a shows splash screen cut between two display monitors

### ISSUE THREAD

**Issue by ferdymercury** [root-project/root#14819]
### Check duplicate issues.

- [ ] Checked for duplicates

### Description

If you have a laptop and an external monitor, the ROOT splash shows inbetween both displays, making it unreadable.

### Reproducer

root -a

### ROOT version

```
ROOT v6.30/04
Built for linuxx8664gcc on Jan 31 2024, 10:01:37
From heads/master@tags/v6-30-04
With c++ (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
Binary directory: /opt/root/bin
```

### Installation method

Binary Release

### Operating system

Ubuntu 22.04

**Comment by ferdymercury** [comment]
See https://github.com/root-project/root/blob/c6b1bce8faad761c5fc13e74698ae6bb075b2a0c/rootx/src/rootxx.cxx#L600

**Comment by guitargeek** [comment]
**Hot take** :fire:

A splash screen is so 90s :laughing: I would just get rid of it entirely. Why is it necessary? The Python interpreter also has none.

At least I would keep it broken like this and *not* fix the issue, so there is one more argument for why it can be removed :slightly_smiling_face:

**Comment by ferdymercury** [comment]
+1 about removing it heheh. The logo is cool though...

**Comment by guitargeek** [comment]
That's true, the logo is nice! But wow, I just took a quick look, and this logo feature is not lightweight at all! It even adds a X11 / cocoa dependency for the ROOT interpreter that would be gone otherwise. I'm checking in a PR what the implications for removing it would be.
